woocommerce/woocommerce-android
# Fix "No Stores" view to reflect design changes

Fixes #3124 by applying the following updates to the "No Woo Stores Found" screen:
- Updated message and added "What is Jetpack" button
- Display the "Learn more about connecting Jetpack" button and make it primary
- Make the "Log in with another account" a secondary button
- Also fixed the margins and centering around the User Info section of the view as it was off-center

Before | After
-- | --
![no-stores-before](https://user-images.githubusercontent.com/5810477/100950511-78da0100-34da-11eb-9d27-2b333e5ffd92.png)|![Screenshot_1606957989](https://user-images.githubusercontent.com/5810477/100950651-ceaea900-34da-11eb-8fc1-3c2b8bfaf6fd.png)


**Landscape version:**

<img src="https://user-images.githubusercontent.com/5810477/100950531-842d2c80-34da-11eb-8454-fdb53b616938.png" width="400"/>

### To Test
1. Click "Continue with WordPress.com" on the first login screen.
2. Log in with a user who has no WooCommerce stores connected to the account. 
3. Click "What is Jetpack" -- verify info dialog appears
4. Click "Learn more about connecting Jetpack" -- verify it loads the Jetpack instructions in a webview
5. Click "Log in with another account" -- verify routed back to the welcome screen. 

Update release notes:
